src/network.c

```diff
@@ -1426,12 +1433,7 @@ static int parse_packet (sockent_t *se, /* {{{ */
 			status = parse_part_encr_aes256 (se,
 					&buffer, &buffer_size, flags);
 			if (status != 0)
-			{
-				ERROR ("network plugin: Decrypting AES256 "
-						"part failed "
-						"with status %i.", status);
```

Comment:
Of course, I still think what developers can be careful enough not to lose ERROR() at parse_part_encr_aes256() but in common cases your approach is much better.

As this type of errors are enough rare (will not count DoS attacks) and users/sysadm can fix that 'easily enough', I agree what your proposal is a good compromise. But there is more complicated case, at least one, see below.

What do you think about logging-level of these messages? 
Some messages (first three at patch) was switched from NOTICE to ERROR level as '-1' ERROR() call proposed to be removed at  upper level in parse_packet().
Should we leave these messages at ERROR level or lower them back to NOTICE?
And what is about log level of rest messages in parse_part_encr_aes256()?

P.S. More complicated case:

`Jun  5 11:37:00 SERVER collectd[24819]: uc_update: Value too old: name = dev/gate-all/gate_octets-all; value time = 1465123010.377; last cache update = 1465123020.376;
Jun  5 11:37:00 SERVER collectd[24819]: Filter subsystem: Built-in target `write': Dispatching value to all write plugins failed with status -1.
Jun  5 11:37:00 SERVER collectd[24819]: Filter subsystem: Built-in target `write': Some write plugin is back to normal operation. `write' succeeded.
`
- network packets come out of order.
- uc_update() generates first line;
- 'rrdtool' plugin has similar check while updating its cache, but silently returns (-1) in that case;
- core treat this as 'plugin failed' and logs second line;
- later metrics are written successfully, so third line appear in logs.;